Cubical/Data/FinSet/Quotients.agda

```diff
@@ -0,0 +1,126 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Data.FinSet.Quotients where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Equiv
+
+open import Cubical.HITs.PropositionalTruncation as Prop
+open import Cubical.HITs.SetQuotients as SetQuot
+open import Cubical.HITs.SetQuotients.EqClass
+
+open import Cubical.Data.Sigma
+
+open import Cubical.Data.FinSet.Base
+open import Cubical.Data.FinSet.Properties
+open import Cubical.Data.FinSet.Constructors
+open import Cubical.Data.FinSet.Cardinality
+
+open import Cubical.Relation.Nullary
+open import Cubical.Relation.Binary
+
+private
+  variable
+    ℓ ℓ' ℓ'' : Level
+
+open Iso
+
+module _
+  (ℓ : Level) where
+
+  DecProp : Type (ℓ-suc ℓ)
+  DecProp = Σ[ P ∈ hProp ℓ ] Dec (P .fst)
+
+  isSetDecProp : isSet DecProp
+  isSetDecProp = isOfHLevelΣ 2 isSetHProp (λ P → isProp→isSet (isPropDec (P .snd)))
```

Comment:
Could these be moved to where `Dec` is defined?

Reply:
Done. I have to create a new file to resolve cyclic dependency, though.

Reply:
Done. I have to create a new file to resolve cyclic dependency, though.